src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -107,30 +104,30 @@ class ContentSidebar extends PureComponent<Props, State> {
     constructor(props: Props) {
         super(props);
         const {
-            cache,
-            token,
-            sharedLink,
-            sharedLinkPassword,
             apiHost,
+            cache,
             clientName,
+            isLarge,
             requestInterceptor,
             responseInterceptor,
+            sharedLink,
+            sharedLinkPassword,
+            token,
         } = props;
 
         this.id = uniqueid('bcs_');
         this.api = new API({
-            cache,
-            token,
-            sharedLink,
-            sharedLinkPassword,
             apiHost,
+            cache,
             clientName,
             requestInterceptor,
             responseInterceptor,
+            sharedLink,
+            sharedLinkPassword,
+            token,
         });
 
-        // Clone initial state to allow for state reset on new files
-        this.state = { ...this.initialState };
+        this.state = { isLoading: true, isOpen: !!isLarge };
```

Comment:
This is fine to set since this component won't unmount while navigating, and we're ok with a reset in behavior on preview close?

Reply:
Just checked with the T3 behavior and it is different from what we have in EUA, we should probably create a ticket to address this in elements and EUA. We'll probably need to expose a `defaultOpen` prop and some sort of toggle callback prop so that EUA can keep track of the state across preview instances

Reply:
What's the issue?

Reply:
If you toggle close the sidebar, and then 'X' out of preview, when you preview another file, the sidebar defaults to open always in EUA, but in T3 it remembers what state it was last

Reply:
Worst case we just store it to session storage

Reply:
Or in the in-memory cache.